File: app/src/main/java/com/example/itlab/authenticationfirebaseapp/Util/GlideModuleStorage.java
package com.example.itlab.authenticationfirebaseapp.Util;

import android.content.Context;

import com.bumptech.glide.Glide;
import com.bumptech.glide.Registry;
import com.bumptech.glide.annotation.GlideModule;
import com.bumptech.glide.module.AppGlideModule;

@GlideModule
public class GlideModuleStorage extends AppGlideModule {

    @Override
    public void registerComponents(Context context, Glide glide, Registry registry) {
        // Register FirebaseImageLoader to handle StorageReference
    }
}

File: android-start/app/src/main/java/com/example/itlab/authenticationfirebaseapp/utils/GlideModuleStorage.kt
package com.example.itlab.authenticationfirebaseapp.utils

import android.content.Context
import com.bumptech.glide.Glide
import com.bumptech.glide.Registry
import com.bumptech.glide.annotation.GlideModule
import com.bumptech.glide.module.AppGlideModule

@GlideModule
class GlideModuleStorage : AppGlideModule() {

    override fun registerComponents(context: Context, glide: Glide, registry: Registry) {
        // Register FirebaseImageLoader to handle StorageReference
    }
}